### DISCUSSION TITLE
[CRITICAL BUG] Firefox on Linux: Pressing CTRL, SHIFT or ALT pops up command pallete when focus is on input fields

### DISCUSSION THREAD

**Discussion by jusko** [community/community#17546] [Other Feature Feedback, Questions, & Ideas]
This _critical_ issue has been affecting some linux firefox users [since March](https://github.community/t/pressing-ctrl-key-only-opens-command-palatte/239883)

Whenever **CTRL**, **SHIFT** or **ALT** are pressed while focus is in a text input/area (even here as I type...) the command palette overlay pops up. 

This destroys capitalization, using alternate characters, deleting/jumping words etc. _**making all text input practically unusable**_.

Using Firefox 101.0.1 on Archlinux. Disabled github shortcuts in accessibility settings (never even considered using the feature as a result... just want old Github back)

**12/06/22 Update:** Please add to this discussion... despite clearly only affecting a minority of users, this is a show stopping problem that needs to be resolved.

**22/06/22 Update:** Primarily due to this, have switched to Brave browser after using FF loyally since its release. :headstone: :disappointed:

**Comment by whyvn** [comment]
I'm using arch and Firefox and i don't have this issue. Do you have any more information or are you doing anything else such as having other programs running, if so; try isolating Firefox and see if you have the same issue.

**Reply by jusko** [reply]
Thanks for your reply. This has been affecting me for ages now (long before this discussion). I've tried disabling all extensions but no luck there.

It's a javascript issue on the site for sure (disabling js stops it) and most likely must be due to Github's command pallete event listeners. I've had a quick look at the callbacks on input fields but they're not responsible.

If anyone has suggestions of where to look I'm happy to debug further when there's some time and feedback the info.

**Reply by whyvn** [reply]
if you've narrowed it down far enough than no one can really help, you just gotta wait until it gets fixed. If you can get more details on the event listener than that might help, sorry.

**Reply by JohnVeness** [reply]
@jusko Testing with extensions disabled is good, but if that hasn't fixed it, try with a completely fresh Firefox profile to see if that does it. Go to about:profiles

**Reply by whyvn** [reply]
You can also test it with Firefox Nightly and make a bug report.
